modelskill/comparison/_collection.py

```diff
@@ -126,6 +126,11 @@ def _add_comparer(self, comparer: Union[Comparer, None]) -> None:
     def name(self) -> str:
         return "Observations"
 
+    @property
+    def unit_text(self) -> str:
+        assert len(set((c.unit_text for c in self))) == 1
```

Comment:
Warn instead? Small inaccuracies in the Quantity could make all the plots fail. And it is not easy to change a Quantity...